src/main.rs

```diff
@@ -67,6 +67,7 @@ quick_main!(run);
 
 fn run() -> cli::error::Result<()> {
 	let version = VersionInfo {
+		name: "Parity ·:· Polkadot",
```

Comment:
Avoid txt designed only for display formatting (`·:·`) here. `Parity Polkadot` will suffice.